internal/mfaportal/resources/frontend/src/pages/authorisation/Oidc.vue

```diff
@@ -1,14 +1,28 @@
 <script setup lang="ts">
+import { useToastError } from '@/composables/useToastError';
 
-setTimeout(function(){
-  window.location.href = "/api/oidc/authorise/start"
-}, 1500)
+const { catcher } = useToastError();
 
+async function doSignIn() {
+  try {
+    window.location.href = "/api/oidc/authorise/start";
+  } catch (e) {
+    catcher(e, "Failed to authorise with IDP");
+  } 
+}
 </script>
 
 <template>
-  <h4 class="card-title text-center">
-    Redirecting you to your SSO provider...
-    <span class="ml-4 loading loading-spinner loading-lg"></span>
-  </h4>
-</template>
+  <div class="auth-card slide-in">
+    <div class="text-center mb-8">
+      <h1 class="text-2xl font-bold">Access your account securely</h1>
+      <p class="text-sm opacity-70">Authenticate with your SSO provider to continue</p>
+    </div>
+
+    <div class="content-section fade-in">
+      <button @click="doSignIn" class="btn btn-primary btn-lg w-full">
+        Sign in with SSO
```

Comment:
This can be "Register with SSO" when you come to doing the registration portion.